apps/aecore/src/aec_accounts.erl

```diff
@@ -13,6 +13,7 @@
 -include("common.hrl").
 -include("trees.hrl").
 
+%% TODO: remove? We should not need such API (init via trees)
```

Comment:
Well spotted - `aec_trees:all_trees_new/1` (called by `aec_block_genesis:genesis_block/0`) shall call this function `aec_accounts:empty/0` rather than `new/0`. (Not crucial of course.)